custom_components/kia_uvo/KiaUvoApiCA.py

```diff
@@ -51,6 +53,7 @@ def __init__(
     def login(self) -> Token:
         username = self.username
         password = self.password
+        pin = self.pin
```

Comment:
you need to store this inside `token.set` as a last variable, which can be used for creating Token instance from json